### DISCUSSION TITLE
Stuck at the test phase

### DISCUSSION THREAD

**Discussion by vincenzocoletta** [keatontaylor/alexa-actions#247] [General]
Hi all, instructions followed with the top of the accuracy for several times (also tried to delete the skill and start again from zero) but unable to pass the test phase. The message I got is something like "There was a problem with the response of the requested skill" (in Italian language "Si è verificato un problema con la risposta della Skill richiesta"). All the suggestions already applied, including the download of the last version of the Lambda but nothing changes... Does anybody know if Claudflare requires the SSL configuration as True? And regards the token and HA address requested into the lambda_function.py code: do they need to be included into quotes like: "https://..." or 'https://...' or nothing _https://..._?
Tks for anything you can suggest!
Regards,
V!

**Comment by DEADSEC-SECURITY** [comment]
Did you import the zip file from the release? Can you send me cloudwatch logs of the skill?
